pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "snakemake-storage-plugin-gcs"
-version = "0.1.3"
+version = "0.1.3.2"
```

Comment:
We usually don't tweak the version - there is a release pipeline for that (sorry missed this before) using release please, here https://github.com/snakemake/snakemake-storage-plugin-gcs/blob/main/.github/workflows/release-please.yml

Reply:
Np, fixed it and added the appropriate commit message that should trigger release please correctly.